### PR TITLE
feat: Add new component - badge

### PR DESCRIPTION
**Pull request by fung-hei-man**
## Description
- Add a new atom component, Badge
- Can be with or without close button

## Scope
- /

### CODE REVIEW

**Review by JiaAnTW — APPROVED** [review]

**Comment by JiaAnTW** [comment]
LGTM 👍

File: resources/assets/js/components/atom/Badge/style.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+import ClearIcon from '@material-ui/icons/Clear';
+import { color } from '~/theme/global';
+
+export const BadgeStyle = styled.span`
+    display: flex;
+    flex-direction: row;
```
Comment (JiaAnTW) [inline resources/assets/js/components/atom/Badge/style.js:7]: sorry剛剛沒注意到，這個好像可以不用?

File: resources/assets/js/components/atom/Badge/style.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    border: none;
+
+    background: ${color.backgroundYellow};
+    color: ${color.darkYellow};
+    font-size: 14px;
+    border-radius: 30px;
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
```
Comment (JiaAnTW) [inline resources/assets/js/components/atom/Badge/style.js:21]: 這三個好像也不需要?

File: resources/assets/js/components/atom/Badge/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { BadgeStyle, CloseIcon } from './style';
+
+function Badge(props) {
```
Comment (JiaAnTW) [inline resources/assets/js/components/atom/Badge/index.js:4]: props 用解構賦值拆開會比較好維護

**Comment by fung-hei-man** [comment]
I was copying the CSS from figma, reviewed & cleared those unused CSS, fixed the style for Chinese (the alignment is different from English...)

Also changed the `props` to params needed.

Please review again, thanks!

**Review by JiaAnTW — APPROVED** [review]

**Comment by JiaAnTW** [comment]
LGTM 👍

**Pull request merged**